foreman-selinux-disable

```diff
@@ -7,11 +7,6 @@ set +e
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
-    # Remove all user defined ports (including the default one)
```

Comment:
We need to remove this on upgrade somehow... maybe move it to the enable script?